srv/components/provisioner/scripts/install.sh

```diff
@@ -28,6 +28,7 @@ repo_url="file:///mnt/cortx"
 nodejs_tar=
 use_local_repo=false
 iso_downloaded_path="/opt/isos"
+MANAGEMENT_IP="$(ip -4 addr show ens32 | grep -oP '(?<=inet\s)\d+(\.\d+){3}')"
```

Comment:
please use small letters for variable name.
Also, can't we use the localhost ip instead of management IP?

Reply:
localhost ip will not work